src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -222,7 +222,7 @@ const Packages = ({ getAllPackages, isSuccess }) => {
   }, [chosenPackages, filterChosen, searchResultsComparator]);
 
   // call api to list available packages
-  const handleAvailablePackagesSearch = async () => {
+  const handleAvailablePackagesSearch = useCallback(async () => {
```

Comment:
I don't understand this change, can you explain it to me please?

Reply:
the usecallback is for better performace, React will reuse the memoized version of the callback and will ensure that only when the dependencis changed it should trigger the recreation of the callback

Reply:
I remove this useCallback from that pr